src/installer.ts

```diff
@@ -604,3 +605,21 @@ export function parseNodeVersionFile(contents: string): string {
 function isLatestSyntax(versionSpec): boolean {
   return ['current', 'latest', 'node'].includes(versionSpec);
 }
+
+export async function enableCorepack(input: string): Promise<void> {
+  let corepackArgs = ['enable'];
+  if (input.length > 0 && input !== 'false') {
+    if (input !== 'true') {
+      const packageManagers = input.split(' ');
+      if (!packageManagers.every(pm => ['npm', 'yarn', 'pnpm'].includes(pm))) {
+        throw new Error(
```

Comment:
Seems like it would be best to let corepack do the validation and have it throw its own error. Imagine when a new package manager is supported in the future.

Reply:
Yeah, I just checked and giving a wrong package manager name will return non 0. So, I guess the `getExecOutput` call will reject the promise on a wrong package, in which case, it should be okay to remove the package manager name check.